content/en/_index.md

```diff
@@ -638,11 +678,15 @@ from the purpose of the snippet.
 
 ### Language compatibility {#lang-compat}
 
+**Applies to:** ✂︎ Snippets, 🧩 Patterns
+
 Use features that work in all Google Cloud-supported versions of a 
```

Comment:
For patterns, I can imagine specific samples (applications) demonstrating specific features that are not compatible Google Cloud wide. I mean, that the goal of a pattern is precisely to demonstrate a corner case feature that applies only to a subset of Google Cloud supported versions etc.

Reply:
Added a note about if an exception is needed to include a code comment explaining.